modules/nf-core/hapibd/meta.yml

```diff
@@ -10,8 +10,8 @@ tools:
       description: "Hap-ibd Detects identity-by-descent (IBD) segments and homozygosity-by-descent (HBD) segments in phased genotype data."
       homepage: "https://github.com/browning-lab/hap-ibd/blob/master/README.md"
       documentation: "https://github.com/browning-lab/hap-ibd/blob/master/README.md"
-      tool_dev_url: "None"
-      doi: "https://doi.org/10.1016/j.ajhg.2020.02.010"
```

Comment:
is the schema seeing the complete url as an error? I think it will be difficult to enforce this structure otherwise.

Reply:
yeah, the pattern is to only have the doi number. Means I don't have to check if it is there when rendering a link on the website.